packages/devtools/web/styles_dark.css

```diff
@@ -66,18 +66,61 @@ nav.menu .menu-item {
     fill-opacity: 0.6 !important;
 }
 
-/* Tweak how buttons look in the dark theme to not be so dark */
-.btn:not(.btn-primary), .btn:not(.btn-primary)[disabled] {
-    border: 1px solid #222222;
-    border-radius: 3px;
-    background: linear-gradient(#5f5f5f 5%, #535353 10%, #4f4f4f 100%);
-    box-shadow: 1px 1px 1px rgba(0,0,0,0.2);
+/* Set defaults for buttons (inc removing Primer styles) */
+.btn, .btn[disabled], .btn:hover, .btn.hover, .btn:active, .btn.selected {
+    background-color: transparent;
+    background-image: none;
+    border-radius: 2px;
```

Comment:
I would either leave the border at 3px like before or update it styles.css instead so the light and dark themes have consistent sizes.
It would be nice is sizes are all identical between the light and dark theme with only colors changing. That way there are never layout bugs in the dark theme only color choices that need to be polished.

Reply:
Done! I think this was an error while I was trying to fix border-widths and I completed it wrong 🙈